app/src/components/DataPanel.vue

```diff
@@ -54,6 +49,17 @@
               @update:comparelayertime="c => comparelayertime = c"
               @compareEnabled="compareEnabled = !compareEnabled"
             /> -->
+            <v-btn
+              ref="regenerateButton"
+              color="secondary"
+              style="display: block;"
+              elevation="2"
+              x-small
+              v-if="customAreaIndicator && showRegenerateButton"
```

Comment:
usually, we have the v-if as the first line in a component, I think it is easier to read the code that way since one already knows in the first line the condition for display